### PR TITLE
Minor updates to general_post

### PR DESCRIPTION
**Pull request by FrankZRS**

### CODE REVIEW

**Comment by FrankZRS** [comment]
~format

File: security/oc_obt.c [hunk 1]
```diff
@@ -3529,6 +3529,19 @@ oc_obt_general_post(oc_uuid_t *uuid, char *query, const char *url,
         } else {
           cbor_encode_text_string(&root_map, "", 0);
         }
+      } else if (strstr(payload_types[i], "bytes") != NULL) {
+        int byte_string_len = (strlen(payload_values[i])  + 1) / 2;
+        unsigned char payload_byte_string[byte_string_len];
```
Comment (WAvdBeek) [inline security/oc_obt.c:3534]: this line does not compile on windows..

File: python/iotivity.py [hunk 2]
```diff
@@ -1157,7 +1158,7 @@ def offboard_all_owned(self):
 
             run_count = 0
             result = False
-            while run_count < 5 and not result: 
+            while run_count < 1 and not result:
```
Comment (Danielius1922) [inline python/iotivity.py:1161]: No need for while, this will now always only run once

Comment (FrankZRS) [reply python/iotivity.py]: Thanks for that!

**Comment by WAvdBeek** [comment]
~format

**Review by WAvdBeek — APPROVED** [review]

**Comment by WAvdBeek** [comment]
~format

**Comment by WAvdBeek** [comment]
not sure why the commit in the format action is not working, merging it anyway, and then fix it later

**Pull request merged**